plugins/buildingplan/buildingplan.cpp

```diff
@@ -44,6 +46,7 @@ const string FILTER_CONFIG_KEY = string(plugin_name) + "/filter";
 const string BLD_CONFIG_KEY = string(plugin_name) + "/building";
 
 static PersistentDataItem config;
+static df::burrow *ignore_burrow = nullptr;
```

Comment:
this seems very dangerous -- what if the burrow is deleted? I think it's better to just keep the id in the config and look it up as necessary -- it could become invalid at any time.